src/template/js/sdk/Transport/mock.js

```diff
@@ -79,11 +79,12 @@ function receive(_callback) {
 }
 
 function event(module, event, value) {
-  const id = Object.entries(event_map).find(([k, v]) => v === module.toLowerCase() + '.' + event.toLowerCase())[0]
-  if (id) {
+  const eventMap = event_map
+  const listener = Object.entries(eventMap).find(([k, v]) => v === module.toLowerCase() + '.' + event)
+  if (listener) {
```

Comment:
This was a bug, if a listener was never set it would return null before the `[0]` lookup. Changed to first check if the listener exists before accessing the id at `listener[0]`